irohad/ametsuchi/impl/temporary_wsv_impl.cpp

```diff
@@ -19,8 +19,6 @@
 #include "ametsuchi/impl/postgres_wsv_command.hpp"
```

Comment:
rearrange includes

Reply:
includes already in the alphabetical order